latency/LatencyScanData.py

```diff
@@ -0,0 +1,86 @@
+#!/bin/env python2.7
```

Comment:
unless you specifically need python 2.7, it's better to just make this ```#!/bin/env python```; as is, it will fail on any machine that doesn't have python 2.7